python-oop/static-methods-and-class-methods.md

```diff
@@ -9,25 +9,260 @@
 
 ## Static Methods are Stateless Methods from a Class
 
+Static methods are methods that are typically called from a class, not an instance.
+
+Additionally, static methods don't have access to instance variables (attributes) or class variables.
+
+Because static methods don't have access to those variables, they tend to be unchanging in nature.
+
+### !callout-info
+
+## Programming Vocab!
+
+The word "static" means "unchanging" and "fixed." The term "static" gets applied to many concepts where the programmer expects it to never change.
+
+### !end-callout
+
 ## Static Methods: Syntax
 
+To define a static method, we:
+
+- use the `@staticmethod` decorator that the Python library provides
+- do not pass in `self` as a parameter
+
+```python
+class ExampleClass:
+
+    @staticmethod
+    def example_method():
+        print("I'm inside the static method, example_method!")
+```
+
+Once a static method is defined, we can invoke it using dot notation, typically from the class itself.
+
+```python
+ExampleClass.example_method()
+```
+
+This code produces the console output:
+
+```
+I'm inside the static method, example_method!
+```
+
+Any code that can reference the class can call a static method. This means that we can call a static method from another static method, class method, instance method, or even inside another class.
+
+### Static Methods: Syntax Details
+
+In Python, we actually _can_ call static methods from an object instance, not just a class. However, because we don't pass in `self`, we don't have a reference to `self`. Static methods do not depend on an instance.
+
+Observe this code, which shows that we _can_ call static methods from an object instance, but we can't access `self`.
+
+```python
+class ExampleClass:
+
+    def __init__(self, name):
+        self.name = name
+
+    @staticmethod
+    def example_method():
+        print("I'm inside the static method, example_method!")
+        print(f"I'm accessing an instance variable, {self.name}")
+```
+
+```python
+example_instance = ExampleClass("Hello, World!")
+
+example_instance.example_method()
+```
+
+This code produces the console output:
+
+```
+I'm inside the static method, example_method!
+Traceback (most recent call last):
+  File "main.py", line 13, in <module>
+    example_instance.example_method()
+  File "main.py", line 9, in example_method
+    print(f"I'm accessing an instance variable, {self.name}")
+NameError: name 'self' is not defined
+```
+
+We should observe from the output that:
+
+- The code _did_ successfully begin the static method, and it _did_ print `I'm inside the static method, example_method!`
+- The code threw a runtime error `NameError`, because we are unable to access `self`
```

Comment:
```suggestion
- The code _did_ successfully begin the static method, and it _did_ print `I'm inside the static method, instanceish_method!`
- The code threw a runtime error `NameError`, because we are unable to access `self` since there is no self parameter.
```